packages/view/backend/sirius-components-view-builder/src/main/java/org/eclipse/sirius/components/view/builder/generated/diagram/SelectionDialogDescriptionBuilder.java

```diff
@@ -43,21 +43,22 @@ public org.eclipse.sirius.components.view.diagram.SelectionDialogDescription bui
     }
 
     /**
-     * Setter for SelectionCandidatesExpression.
+     * Setter for SelectionMessage.
      *
      * @generated
      */
-    public SelectionDialogDescriptionBuilder selectionCandidatesExpression(java.lang.String value) {
-        this.getSelectionDialogDescription().setSelectionCandidatesExpression(value);
+    public SelectionDialogDescriptionBuilder selectionMessage(java.lang.String value) {
```

Comment:
Same comment

Reply:
I was mistaken, it has already been rebased on top of the commit I was thinking about. There's nothing to change here